src/Pages/SignUp/Signup.scss

```
@@ -0,0 +1,82 @@
+.signup_form{
+  padding:20px 15px;
+  margin: 30px;
+  .signup_container{
+    border: 1px solid #DEDEDE;
+    margin-bottom: 15px;
+    .signup_header{
+      background-color: #DEDEDE;
+      display: flex;
+      height: 40x;
+      .text_general{
+        background: #ADC088;
+        padding: 0 8px;
+        line-height: 40px;
+      }
```

```suggestion
.signupForm {
  padding:20px 15px;
  margin: 30px;
  
  .signupContainer {
    border: 1px solid #DEDEDE;
    margin-bottom: 15px;
    
    .signupHeader {
      background-color: #DEDEDE;
      display: flex;
      height: 40x;
      
      .textGeneral {
        background: #ADC088;
        padding: 0 8px;
        line-height: 40px;
      }
```
- 새로운 셀렉터 앞에서는 한줄 띄워주세요
- 카멜케이스로 클래스명 통일시켜주세요
- 클래스명 다음 한칸 띄우고 중괄호 시작할 수 있게 해주세요

---

네~